src/utils/validateTrimmedString.ts

```diff
@@ -1,10 +1,41 @@
-// makes sure the string passed into the function isn't an empty string
-export function validateTrimmedString(input: string) {
-	const trimmedInput = input.trim();
+import { ListItem } from "../api";
 
+// Validates the item name input for a shopping list
+export function validateItemName(
+	input: string,
+	existingItems: ListItem[],
+): string | null {
+	const trimmedInput = input.trim(); //removes leading and trailing whitespaces
+
+	// Condition 1: Check if the input is empty
 	if (trimmedInput.length === 0) {
-		return null;
+		return "Item cannot be empty";
+	}
+
+	//Remove punctuation marks and normalize input
+	const punctuationRegex = /[^\p{L}]/gu;
```

Comment:
praise: love the use of the regex, it is something I'm still such a novice to but I know it has a lot of good functionality and it is awesome to see it here!

Reply:
It was really awesome to bring it back to life form the dusty attic! lol

Reply:
alternate regex option: `input.toLowerCase().replace(/[^a-z0-9]/g, '')`